packages/core/__test__/sociable/sociable-testbed-builder.integration.test.ts

```diff
@@ -0,0 +1,122 @@
+import { StubbedInstance } from '@suites/types.doubles';
+import { FakeAdapter } from './assets/integration.assets';
+import { mock } from './assets/mock.static';
+import {
+  Logger,
+  Repository,
+  UserApiService,
+  UserDal,
+  UserService,
+  UserVerificationService,
+  ApiService,
+  DatabaseService,
+  User,
+} from './assets/injectable-registry.fixture';
+import { UnitReference } from '@suites/unit';
+import { SociableTestBedBuilder, UnitMocker } from '../../src';
+import * as console from 'console';
+
+describe('Social TestBed Builder Integration Tests', () => {
+  let unitBuilder: SociableTestBedBuilder<UserService>;
+  let userServiceAsIfItWasUnderTest: UserService;
+  let unitRef: UnitReference;
+
+  beforeAll(async () => {
+    unitBuilder = new SociableTestBedBuilder(
+      Promise.resolve(mock),
+      Promise.resolve(FakeAdapter),
+      new UnitMocker(Promise.resolve(mock), Promise.resolve(FakeAdapter)),
+      UserService,
+      console
+    );
+
+    const testBed = await unitBuilder
+      .expose(UserApiService)
```

Comment:
Because this is a public user-facing API, lets discuss the chosen terminology expose, to mark a dependency to not be faked.

What is the mental model Automock tries to give its users about what dependencies get replaced with test-doubles and what aren't?

In my mind for example, since Automock uses the terminology: "Automock, mock...using", I'd expect this to be called dontMock or something like that.

Reply:
@potoos @qballer what do you think?